helpers/sql-obfuscation/opentelemetry-helpers-sql-obfuscation.gemspec

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/helpers/sql_obfuscation/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-helpers-sql-obfuscation'
+  spec.version     = OpenTelemetry::Helpers::SqlObfuscation::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'SQL Obfuscation Instrumentation Helpers for the OpenTelemetry framework'
+  spec.description = 'SQL Obfuscation Instrumentation Helpers for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = Dir.glob('lib/**/*.rb') +
+               Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 3.0'
+
+  spec.add_dependency 'opentelemetry-sdk', '~> 1.0'
```

Comment:
This should be an API dependency and must not depend on the SDK. 

`common` and `semenatic_conventions` are also ok.

Reply:
I'll update this to depend on `common`, that's the specific package we need.

Reply:
Fixed in: 891227424747422b236d2cc91ba1a419ec4ff21c